### PR TITLE
Fix search

### PR DESCRIPTION
**Pull request by emumoo**
hi @aw3as

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -245,3 +245,34 @@ select {
...
+@media (max-width: $screen-sm) {
+    .navbar-default .navbar-brand {
+      background-color: #fff;
+    }
+    #logo {
+      margin-left: -30px;
+    }
+}
+
+.navbar-toggle {
```
Comment (aw3as) [inline app/assets/stylesheets/application.scss]: what is this css for?

File: app/assets/stylesheets/courses.scss [hunk 2]
```diff
@@ -4,6 +4,10 @@ h1 {
   font-weight: 300;
 }
 
+.title-link:hover {
```
Comment (aw3as) [inline app/assets/stylesheets/courses.scss]: is this still needed?

File: app/assets/stylesheets/professor.scss [hunk 3]
```diff
@@ -1,3 +1,45 @@
 @import "variables";
 
+#professor-header {
```
Comment (aw3as) [inline app/assets/stylesheets/professor.scss]: is this for the professor page?

File: app/views/professors/show.html.slim [hunk 4]
```diff
@@ -1,22 +1,29 @@
...
-ul.professor-courses style="list-style-type: none; padding: 15px;"
-  - mnemonic = @courses.first.subdepartment.mnemonic
-  - @courses.each do |c|
-    - if c.subdepartment.mnemonic != mnemonic
-      - mnemonic = c.subdepartment.mnemonic
-      br
-    li= link_to "#{c.mnemonic_number} #{c.title}", course_path(c.id, :p => @professor.id)
\ No newline at end of file
+
+#professor-header
```
Comment (aw3as) [inline app/views/professors/show.html.slim]: this entire page needs to look better

Comment (emumoo) [inline app/views/professors/show.html.slim]: I agree

File: app/assets/stylesheets/professor.scss (hunk 3 above)
Comment (emumoo) [inline app/assets/stylesheets/professor.scss]: yus

File: app/views/courses/show.html.slim [hunk 5]
```diff
@@ -1,6 +1,9 @@
 - content_for :page_heading, "#{@course.mnemonic_number} - #{@course.title}"
+- content_for :course_link, "/courses/#{@course.id}/professors"
```
Comment (aw3as) [inline app/views/courses/show.html.slim]: not apparent that you can click on this

File: app/views/courses/show.html.slim [hunk 6]
```diff
@@ -1,6 +1,9 @@
...
+
 - if @professor
   - content_for :course_professor, "#{@professor.full_name}"
+  - content_for :professor_link, "/professors/#{@professor.id}"
```
Comment (aw3as) [inline app/views/courses/show.html.slim]: nor this

File: app/assets/stylesheets/courses.scss (hunk 2 above)
Comment (emumoo) [inline app/assets/stylesheets/courses.scss]: To prevent underlining on hover, it looks bad

**Pull request merged**